jms/1.7/modules/ROOT/pages/jms-studio-configuration.adoc

```diff
@@ -0,0 +1,473 @@
+
+[[configure-connection]]
+== Configure the Connection to a Broker
+
+To configure the connection in the global element, define the connection by either selecting *ActiveMQ Connection* or *Generic Connection*, set up the connection libraries and the JMS specification. +
+JMS Connector also enables you to provide credentials if you require an authenticated connection, and also to configure connections caching to increase the performance of the application.
+
+=== Configure an Active MQ Connection
+
+JMS Connector supports ActiveMQ 5 by using the *ActiveMQ Connection*. With this connection, you can use both `JMS_1_1 (Default)` or `JMS_1_0_2b` specifications, and configure all the general connection parameters for JMS, as well as the custom parameters only present in ActiveMQ.
+
+Once you declare the ActiveMQ connection, set up the *Connection Factory* with your custom configuration. Every parameter in the connection comes with a default value, meaning that you are required to configure only the parameters relevant for your use case. Also, the ActiveMQ connection exposes parameters that are exclusive of the ActiveMQ implementation, like *Initial redelivery delay*.
+
+In the following example you configure an *ActiveMQ Connection*:
+
+. In Studio, navigate to the *Global Elements* tab.
+. Click *Create*.
+. In the filter box type `jms`, and select *JMS Config*.
+. Click *OK*.
+. In the *Connection* field select *ActiveMQ Connection*.
+. In the *Factory configuration* field select *Edit Inline*.
+. Set the *Broker url* field to address of the broker to connect, for example `tcp://localhost:61616`.
+. Click *OK*.
+
+.JMS Connector ActiveMQ Configuration
+image::jms-activemq-configuration.png[ActiveMQ Connection Configuration.]
+
+In the XML editor, the `<jms:active-mq-connection>` and `<jms:factory-configuration>` configuration looks like this:
+
+[source,xml,linenums]
+----
+ <jms:config name="JMS_Config">
+  <jms:active-mq-connection >
+   <jms:factory-configuration brokerUrl="tcp://localhost:61616" />
+  </jms:active-mq-connection>
+ </jms:config>
+----
+
+=== Configure a Generic Connection
+
+Use the *Generic Connection* to configure a connection to any broker implementation. JMS Connector provides a JNDI (Java Naming and Directory Interface) based connection factory builder that enables you to configure the connection using JNDI in the context of the application.
+
+In the following example, you configure a *Generic Connection* to Artemis to use the `JMS_2_0` specification:
+
+. In Studio, navigate to the *Global Elements* tab.
+. Click *Create*.
+. In the filter box type `jms`, and select *JMS Config*.
+. Click *OK*.
+. In the *Connection* field select *Generic Connection*.
+. In the *Specification* field select *JMS_2_0*.
+. In the *Connection factory* field select *Edit inline*.
+. Set the *Connection factory jndi name* field to `ConnectionFactory`.
+. In the *Lookup destination* field select any of the following options:
++
+* *NEVER* +
```

Comment:
Even though the way you have this list formatted looks way better in the output and is correct, Kim will complain and tell you to put the + on the next line.